packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -449,6 +449,159 @@ curl -v -X GET \
 ]
 ```
 
+### List YubiKey OTP Tokens
+
```

Comment:
Can we add this as well between the header and description?

`<ApiOperation method="get" url="/api/v1/org/factors/yubikey_token/tokens" />`